r/DESCRIPTION

```diff
@@ -40,9 +40,11 @@ Roxygen: list(markdown = TRUE, r6 = FALSE, load = "source")
 RoxygenNote: 7.1.1.9001
 VignetteBuilder: knitr
 Suggests:
+    DBI,
     decor,
     distro,
     dplyr,
+    duckdb (> 0.2.7),
```

Comment:
Do we need to suggest dbplyr too?

Reply:
For the tests to run, yes. For now I added it as a step in the CI setup, since these tests _are_ pretty peripheral and for ~anyone installing any of the packages for real I'm certain they will already have dbplyr installed. But we can add it to suggests if we don't think that's too heavy.